### PR TITLE
ADM-831:[frontend]feat: add pipeline is future time check

### PR DESCRIPTION
**Pull request by weiraneve**
## Summary

...

## Before

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## After

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## Note

_Null_

### CODE REVIEW

**Review by SimonTal — APPROVED** [review]

File: frontend/src/containers/MetricsStep/index.tsx [hunk 1]
```diff
@@ -60,6 +66,7 @@ const MetricsStep = () => {
   const { getBoardInfo, isLoading, errorMessage } = useGetBoardInfoEffect();
   const shouldLoad = useAppSelector(shouldMetricsLoad);
   const shouldGetBoardConfig = useAppSelector(selectShouldGetBoardConfig);
+  const isFutureTime = !!startDate && new Date(startDate) > new Date();
```
Comment (guzhongren) [inline frontend/src/containers/MetricsStep/index.tsx:69]: use dayjs to compare, ref to https://day.js.org/docs/en/query/is-before

File: frontend/__tests__/containers/MetricsStep/MetricsStep.test.tsx [hunk 2]
```diff
@@ -304,6 +304,26 @@ describe('MetricsStep', () => {
       ).toBeInTheDocument();
     });
 
+    it('should be render no pipeline container when get pipeline when selected future time ', async () => {
+      const MOCK_DATE_RANGE = {
```
Comment (guzhongren) [inline frontend/__tests__/containers/MetricsStep/MetricsStep.test.tsx:308]: Should not mock coming dateTime, should mock current time to an old dateTime & select `mocked current dateTime` <`the old dateTime` < `real now()`

File: frontend/__tests__/containers/MetricsStep/MetricsStep.test.tsx [hunk 3]
```diff
@@ -304,6 +304,26 @@ describe('MetricsStep', () => {
       ).toBeInTheDocument();
     });
 
+    it('should be render no pipeline container when get pipeline when selected future time ', async () => {
```
Comment (guzhongren) [inline frontend/__tests__/containers/MetricsStep/MetricsStep.test.tsx:307]: `should render no pipeline.....`

File: frontend/src/constants/resources.ts [hunk 4]
```diff
@@ -284,6 +284,11 @@ export const BOARD_CONFIG_INFO_TITLE = {
   EMPTY: '',
 };
 
+export const PIPELINE_CONFIG_INFO_TITLE = {
```
Comment (guzhongren) [inline frontend/src/constants/resources.ts:287]: this variable like an `enum`, please covert it to an enum, and rename it as `PipelineConfigInfoTitle`

ref to https://zhuanlan.zhihu.com/p/88609380, same as JS

**Review by guzhongren — COMMENTED** [review]
Change with the comments

**Pull request merged**